docs/ch12-qtquick3d/assets.md

```diff
@@ -0,0 +1,106 @@
+# Working with Assets
+
+When working with 3D scenes, the built in meshes quickly grow old. Instead, a good flow from a modelling tool into QML is needed. Qt Quick 3D comes with the Balsam asset import tool, which is used to convert common asset formats into a Qt Quick 3D friendly format.
+
+The purpose of Balsam is to make it easy to take assets created in common tools such as [Blender](https://www.blender.org/), Maya or 3ds Max and use them from Qt Quick 3D. Balsam supports the following asset types:
+
+- COLLADA (``*.dae``)
+- FBX (``*.fbx``)
+- GLTF2 (``*.gltf``, ``*.glb``)
+- STL (``*.stl``)
+- Wavefront (``*.obj``)
+
+For some format, texture assets can also be exported into a Qt Quick 3D-friendly format, as long as Qt Quick 3D supports the given asset.
+
+::: tip Tip
+Balsam also supports converting Qt 3D Studio projects (``*.uia``, ``*.uip``) to Qt Quick 3D. As the feature set is not identical, only the features supported by Qt Quick 3D can be converted.
```

Comment:
Remove this "Tip".  We are likely to remove this functionality soon.  It doesn't really work in any meaningful way at this point.

Reply:
Done!